locales/en-US/editor.properties

```diff
@@ -218,6 +218,12 @@ PIXEL_UNIT=pixels
 PERCENTAGE_UNIT=percentage
 EM_UNIT=em font unit
 
+# extensions/default/InlinePaddingEditor
+
+ALL_SIDE=All Sides
```

Comment:
can we use more meaningful variable names or maybe add a comment above each of these explaining how they will be used?